README.md

```diff
@@ -423,6 +423,7 @@ See also [A comparison of operating systems written in Rust](https://github.com/
   * [cargo-ebuild](https://crates.io/crates/cargo-ebuild) — cargo extension that can generate ebuilds using the in-tree eclasses [<img src="https://api.travis-ci.org/cardoe/cargo-ebuild.svg?branch=master">](https://travis-ci.org/cardoe/cargo-ebuild)
   * [cargo-edit](https://crates.io/crates/cargo-edit) — allows you to add and list dependencies by reading/writing to your Cargo.toml file from the command line [<img src="https://api.travis-ci.org/killercup/cargo-edit.svg?branch=master">](https://travis-ci.org/killercup/cargo-edit)
   * [cargo-find](https://crates.io/crates/cargo-find) <sup>deprecated</sup> — Find crates from command line [<img src="https://api.travis-ci.org/Ralvke/cargo-find.svg?branch=master">](https://travis-ci.org/Ralvke/cargo-find)
+  * [cargo-generate](https://github.com/ashleygwilliams/cargo-generate) A generator of a rust project by leveraging a pre-existing git repository as a template [<img src="https://api.travis-ci.org/ashleygwilliams/cargo-generate.svg?branch=master">](https://travis-ci.org/ashleygwilliams/cargo-generate)
```

Comment:
This refers to a travis build on `travis-ci.org` but the builds appear to be on `travis-ci.com`. Could you fix this please?

Reply:
Oops, thank you for your reviewing! I'll fix it.

Reply:
Fixed it; rebased with fixup!